src/lib/CiaoData.h

```diff
@@ -36,18 +39,18 @@
 class CiaoData {
 	public:
 
-		char* get(int index){
+		const char* msg_split[3];
+
+		const char* get(int index){
 			return msg_split[index];
 		}
 
 		bool isEmpty(){
 			//if (atoi( get(1) ) > 0)
-			if (get(2) != "")
+			//if (get(2) != "")
+			if ( strcmp(get(2), "") != 0)
```

Comment:
`strcmp` is unsafe, it's better if you use `strncmp` (or something even easier, based on what is the expected result of `get(2)`)